### DISCUSSION TITLE
Request for face recognition distillation example

### DISCUSSION THREAD

**Discussion by thanhnt-3728** [yoshitomo-matsubara/torchdistill#156] [General]
Do you have any plan for sharing face recognition code to reproduce results in https://arxiv.org/pdf/1904.01802v1.pdf
Thank you

**Comment by yoshitomo-matsubara** [comment]
Hi @thanhnt-2658 

I think you can run the experiment with [`image_classification.py` ](https://github.com/yoshitomo-matsubara/torchdistill/tree/main/examples), using [this CCKD config file](https://github.com/yoshitomo-matsubara/torchdistill/blob/main/configs/sample/ilsvrc2012/single_stage/cckd/resnet18_from_resnet50.yaml).
You will mainly edit the `datasets` and `models` entries, and if you can find and prepare models and datasets, I can help you run the experiment.

Note that the CCKD config does not consider their sampling part in the paper as it looks not described enough to be reimplemented. (e.g., which feature in teacher model they used).
